packages/react-native/src/primitives/Checkbox/Checkbox.tsx

```diff
@@ -7,11 +7,8 @@ import {
 } from 'react-native';
 
 import { icons } from '../../assets';
-import {
-  IconButton,
-  Label,
-} from '@aws-amplify/ui-react-native/dist/primitives';
-
+import { IconButton } from '../IconButton';
+import { Label } from '../Label';
 import { CheckboxProps, CheckboxStyle } from './types';
 
 const styles: CheckboxStyle = {
```

Comment:
Should we put these styles in a separate file to match the pattern for other primitives in the RN package?